src/main/resources/org/openstreetmap/atlas/utilities/command/subcommands/OsmToAtlasCommandExamplesSection.txt

```diff
@@ -0,0 +1,10 @@
+atlas osm2Atlas zz.osm
+    converts to zz.atlas in the current working directory
```

Comment:
Thanks for putting so many examples. This is awesome stuff. A quick note on what the `man` page generator expects. If you want the generator to parse it properly, try something like this:
```
Converts to zz.atlas in the current working directory:
#atlas osm2Atlas zz.osm --josm
Converts to zz.atlas in the current working directory where zz.osm is saved from josm:
#atlas osm2Atlas zz.osm --josm
etc.
```
The leading `#` will tell the `man` page generation code to indent and highlight the following line as a code block. The above doc file would be formatted in the man page as:
```
Converts to zz.atlas in the current working directory:
    
    atlas osm2Atlas zz.osm --josm

Converts to zz.atlas in the current working directory where zz.osm is saved from josm:

    atlas osm2Atlas zz.osm --josm
```

Without the `#`, newlines and extra spaces are ignored. This is because the man page generator explicitly ignores your whitespace directives in order to support arbitrary terminal widths (try viewing the `--help` page with different sized terminals, you will see it always chooses a sane formatting). However, the `#` overrides this behavior for the current line.